topics/sequence-analysis/tutorials/annotation-with-prokka/tutorial.md

```diff
@@ -0,0 +1,139 @@
+
+## Learning objectives
+{:.no_toc}
+
+At the end of this tutorial you should be able to:
+
+1. load a genome assembly into Prokka
+2. annotate the assembly using Prokka
+3. examine the annotated genome using JBrowse
+
+> ### Agenda
+>
+> In this tutorial, we will deal with:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+## Import the data
+
+Prokka requires assembled contigs.
+
+> ### {% icon hands_on %} Hands-on: Obtaining our data
+>
+> 1. Make sure you have an empty analysis history. Give it a name.
+>
+>    > ### {% icon tip %} Starting a new history
+>    >
+>    > * Click the **gear icon** at the top of the history panel
+>    > * Select the option **Create New** from the menu
+>    {: .tip}
+>
+> 2. **Import Sample Data.** The data for this course may be available from a shared library in Galaxy
+> (ask your instructor). If this is not the case, you can upload it yourself.
+> - Option 1: From data library:
+>   - Navigate to the shared data library, you should find a file called `contigs.fasta`
+> - Option 2: From your computer:
+>   - Obtain data directly from Zenodo: [![DOI](https://zenodo.org/badge/DOI/10.5281/zenodo.1156405.svg)](https://doi.org/10.5281/zenodo.1156405)
+>   - Download `contigs.fasta`
+>   - Upload the file to your history.
+> <br><br>
+>
+{: .hands_on}
+
+## Annotate the genome
+
+Now we will run the tool called Prokka.
+
+> ### {% icon hands_on %} Hands-on: Annotate genome
+>
+> 1. **Prokka** {% icon tool %} with the following parameters (leave everything else unchanged)
+>    - contigs to annotate: `contigs.fasta`
+>    - Locus tag prefix (--locustag): P
+>    - Force GenBank/ENA/DDJB compliance (--compliant): No
+>    - Sequencing Centre ID (--centre): V
+>    - Genus Name: Staphylococcus  
+>    - Species Name: aureus  
+>    - Use genus-specific BLAST database: No  
+>    - Your tool interface should look like this:
+>    - ![prokka interface](images/interface.png)
+>    - Click Execute
+> <br><br>
+{: .hands_on}
+
+## Examine the output
+
+Once Prokka has finished, examine each of its output files.
+
+> ### {% icon hands_on %} Hands-on: output files
+>
+> - The GFF and GBK files contain all of the information about the features annotated (in different formats.)
+> - The .txt file contains a summary of the number of features annotated.
+> - The .faa file contains the protein sequences of the genes annotated.
+> - The .ffn file contains the nucleotide sequences of the genes annotated.
+> <br><br>
+{: .hands_on}
+
+## View annotated features in JBrowse
+
+Now that we have annotated the draft genome sequence, we would like to view the sequence in the JBrowse genome viewer. First, we have to make a JBrowse file. Then, we can view it within Galaxy.
+
+> ### {% icon hands_on %} Hands-on: Visualize annotation
```

Comment:
What do you think about simplifying here and having something I proposed in #667 : https://github.com/galaxyproject/training-material/pull/667/files/39c6ea6487e244b2c023213d5925c4113db8199d#diff-85eb4ab60195721a80f9728d9a3ac220R150

![screen shot 2018-01-23 at 15 00 11](https://user-images.githubusercontent.com/1842467/35281333-f6ad0970-0052-11e8-8211-7c6480d91dc1.png)
![screen shot 2018-01-23 at 15 00 23](https://user-images.githubusercontent.com/1842467/35281334-f6ea5870-0052-11e8-89b9-023b08a1b85a.png)

Reply:
@bebatut I think that's a great idea. I've removed the screenshot of the tool parameters and changed the formatting to match your example.